importer/tag/src/main/java/org/itsallcode/openfasttrace/importer/tag/RegexLineConsumer.java

```diff
@@ -36,10 +36,11 @@ public void readLine(final int lineNumber, final String line)
      * @param matcher
      *            regular expression {@link Matcher}.
      * @param lineNumber
-     *            line number of the matched the input, starting with  {@code 1} for the first line.
+     *            line number of the matched the input, starting with {@code 1}
+     *            for the first line.
      * @param lineMatchCount
-     *            number of the current match in the context of the current line,
-     *            starting with {@code 0} for the first match in a line.
+     *            number of the current match in the context of the current
```

Comment:
Strange. Line width for comments and code should be 100 chars. Why it this wrapping here?